<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myCrypt Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="inputField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>561</width>
      <height>111</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>561</width>
      <height>121</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QSlider" name="shiftSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="shiftAmountLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Shift Amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cryptType">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>10</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Caesar Cipher</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sliderValueLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cryptDropDown">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Caesar Cipher</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Caesar Cipher</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vigenere</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="keyInputField">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="centerOnScroll">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="keyLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
